### PR TITLE
Pset_EarthworksCutCommon

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_EarthworksCutCommon

## Description: 

Common properties for earthworks cut elements

| Property                      | Datatype                            | Description                           |
| ----------------------------- | ----------------------------------- | ------------------------------------- |
| EarthworksCutExcavationMethod | PEnum_EarthworksCutExcavationMethod | Method of excavation                  |
| UserDefinedExcavationMethod         | IfcLabel                            | Custom or specialized excavation methods beyond standard categories. Used when EarthworksCutExcavationMethod is set to USERDEFINED. |
| MaximumUnsupportedCut         | IfcNonNegativeLengthMeasure         | Maximum unsupported depth of cut      |
| SupportDescription            | IfcLabel                            | Description of Support                |

## Enumeration: PEnum_EarthworksCutExcavationMethod
Enumeration of earthworks cut methods.

| Constant      | Description |
| ------------- | ----------- |
| HAND          |   Manual excavation method using hand tools.   |
| EXCAVATOR     |   Mechanized method employing excavator machinery.  |
| JACKHAMMER    |   Pneumatic or hydraulic tool breaking rock or concrete.   |
| RIPPER        | Equipment with a toothed tool to break and loosen hard soil. |
| DRILLANDBLAST | Controlled drilling followed by explosive blasting.  |
| USERDEFINED         | Existing enum constant  |

### CODE REVIEW

File: IFC4x3/Properties/m/MaximumUnsupportedCut_103uAEL3z7QhsU6g8irstN/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MaximumUnsupportedCut_103uAEL3z7QhsU6g8irstN/DocProperty.xml]: Did we not have this property somewhere already?

File: IFC4x3/Properties/s/SupportDescription_01gbdSFfjCmwU9fHFwjDV5/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SupportDescription_01gbdSFfjCmwU9fHFwjDV5/DocProperty.xml]: Also seen this somewhere already.

**Review by SergejMuhic — COMMENTED** [review]
Descriptions of constants and enums.

File: IFC4x3/Properties/m/MaximumUnsupportedCut_103uAEL3z7QhsU6g8irstN/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/m/MaximumUnsupportedCut_103uAEL3z7QhsU6g8irstN/DocProperty.xml]: Can't find in the current spec?

File: IFC4x3/Properties/s/SupportDescription_01gbdSFfjCmwU9fHFwjDV5/DocProperty.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/s/SupportDescription_01gbdSFfjCmwU9fHFwjDV5/DocProperty.xml]: Can't find in the current spec?

File: IFC4x3/Properties/m/MaximumUnsupportedCut_103uAEL3z7QhsU6g8irstN/DocProperty.xml (hunk 1 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/m/MaximumUnsupportedCut_103uAEL3z7QhsU6g8irstN/DocProperty.xml]: In that case, I merge and we try on the merged spec afterwards.

File: IFC4x3/Properties/s/SupportDescription_01gbdSFfjCmwU9fHFwjDV5/DocProperty.xml (hunk 2 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/s/SupportDescription_01gbdSFfjCmwU9fHFwjDV5/DocProperty.xml]: In that case, I merge and we try on the merged spec afterwards.

**Comment by SergejMuhic** [comment]
Just that `OTHER` constant in general I guess that I mentioned [here](https://github.com/bSI-InfraRoom/IFC-Specification/pull/564#pullrequestreview-1748790820).

Is it `OTHER` that we want to go for or the standard IFC property enum constants?

**Comment by larswik** [comment]
ping Ping @JontyDunn and @hessf-ilf. Can you describe the enums?

**Comment by larswik** [comment]
> Just that `OTHER` constant in general I guess that I mentioned [here](https://github.com/bSI-InfraRoom/IFC-Specification/pull/564#pullrequestreview-1748790820).
> 
> Is it `OTHER` that we want to go for or the standard IFC property enum constants?

Switch to USERDEFINED then?

**Comment by hessf-ilf** [comment]
> ping Ping @JontyDunn and @hessf-ilf. Can you describe the enums?

Suggestion:
1. **HAND:**
   - Manual excavation method using hand tools.

2. **EXCAVATOR:**
   - Mechanized method employing excavator machinery.

3. **JACKHAMMER:**
   - Pneumatic or hydraulic tool breaking rock or concrete.

4. **RIPPER:**
   - Equipment with a toothed tool to break and loosen hard soil.

5. **DRILLANDBLAST:**
   - Controlled drilling followed by explosive blasting.

6. **OTHER:**
   - Custom or specialized excavation methods beyond standard categories.

**Comment by larswik** [comment]
> > Just that `OTHER` constant in general I guess that I mentioned [here](https://github.com/bSI-InfraRoom/IFC-Specification/pull/564#pullrequestreview-1748790820).
> > Is it `OTHER` that we want to go for or the standard IFC property enum constants?
> 
> Switch to USERDEFINED then?

Fixed in [f372361](https://github.com/bSI-InfraRoom/IFC-Specification/pull/571/commits/f372361684d8ad2e453a5ca9ed5b51cad6b74e6d)

**Comment by larswik** [comment]
> Descriptions of constants and enums.

See [f372361](https://github.com/bSI-InfraRoom/IFC-Specification/pull/571/commits/f372361684d8ad2e453a5ca9ed5b51cad6b74e6d)

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**